modules/core/src/v2/coins/eth.ts

```diff
@@ -832,6 +845,45 @@ export class Eth extends BaseCoin {
     }).call(this).asCallback(callback);
   }
 
+  /**
+   * Return types, values, and total amount to send in a batch transaction
+   * @param recipients {Object} transaction recipients
+   * @returns {Object} the types, values, and total amount to execute the batch transaction
+   */
+  static batchExecutionInfo(recipients) {
```

Comment:
Also please add a type for args and return values (see interfaces at the top of this file for how to quickly define a type)

Reply:
Fixed.